src/views/viewer/ChannelView/ChannelView.style.tsx

```diff
@@ -175,7 +175,7 @@ type SearchContainerProps = {
 export const SearchContainer = styled.div<SearchContainerProps>`
   display: flex;
   grid-area: search;
-  width: 100%;
+  width: 192px;
```

Comment:
Why are we making this change to channel view? Seems outside of scope of this PR

Reply:
It's a fix I added in every PR after fixing the conflicts of the Search component, not ideal but it won't matter as it will be in discovery as well.

Reply:
Yes, I think I was confused about this, removed the change